tests/regression/lp-1848567/task.yaml

```diff
@@ -21,9 +21,9 @@ execute: |
     # Re-compile the apparmor profile for snap-update-ns for the
     # test-snapd-app snap while ensuring that the profile is not loaded
     # into kernel memory and that the compiler is not using any existing
-    # caches. Use memory-tool(1) to record the maximum resident memory usage
```

Comment:
there was even a manpage?

Reply:
No, I wrote some man pages but after failing to use them in any way (setting MANPATH and stuff) I never proposed them. I guess I just grew a habit of writing this way.